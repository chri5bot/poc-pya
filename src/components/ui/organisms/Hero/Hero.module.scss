.hero {
  align-items: center;
  background: black;
  display: flex;
  height: 100vh;
  justify-content: center;
  text-align: center;
  width: 100vw;

  h1 {
    color: white;
    font-size: 4rem;
  }

  h2 {
    color: white;
    font-size: 2rem;
    font-style: italic;
  }
}
