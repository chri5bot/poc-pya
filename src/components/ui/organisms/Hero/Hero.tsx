import styles from './Hero.module.scss';

export interface HeroProps {
  codename: string;
}

function Hero({ codename }: HeroProps) {
  return (
    <div className={styles.hero}>
      <div className={styles.content}>
        <h2>{codename}</h2>
      </div>
    </div>
  );
}

export default Hero;
