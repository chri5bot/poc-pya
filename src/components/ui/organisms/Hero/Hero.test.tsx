import React from 'react';
import { shallow } from 'enzyme';

import Hero from './Hero';

describe('Organisms/Hero', () => {
  def('props', () => ({ codename: 'test' }));

  def('subject', () => {
    const wrapper = shallow(<Hero {...$.props} />);

    return toJSON(wrapper);
  });

  it('renders hero with codename', () => {
    expect($.subject).toMatchSnapshot();
  });
});
