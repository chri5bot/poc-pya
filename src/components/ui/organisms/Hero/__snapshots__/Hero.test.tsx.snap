// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Organisms/Hero renders hero with codename 1`] = `
<div
  className="hero"
>
  <div
    className="content"
  >
    <h2>
      test
    </h2>
  </div>
</div>
`;
